graphprot/DataSet.py

```diff
@@ -221,51 +221,52 @@ def load_one_graph(self, fname, mol):
-                data += (vals,)
-            data = np.hstack(data)
-            data = np.vstack((data, data))
-            data = self.edge_feature_transform(data)
-            edge_attr = torch.tensor(
-                data, dtype=torch.float).contiguous()
-
-        else:
-            edge_attr = None
-
-        # internal edges
-        ind = grp['internal_edge_index'][()]
-        ind = np.vstack((ind, np.flip(ind, 1))).T
-        internal_edge_index = torch.tensor(
-            ind, dtype=torch.long).contiguous()
-
-        # internal edge feature
-        data = ()
-        if self.edge_feature is not None:
-            for feat in self.edge_feature:
-                vals = grp['internal_edge_data/'+feat][()]
-                if vals.ndim == 1:
-                    vals = vals.reshape(-1, 1)
-                data += (vals,)
-            data = np.hstack(data)
-            data = np.vstack((data, data))
-            data = self.edge_feature_transform(data)
-            internal_edge_attr = torch.tensor(
-                data, dtype=torch.float).contiguous()
-
-        else:
-            internal_edge_attr = None
-
+        try:
+            # index ! we have to have all the edges i.e : (i,j) and (j,i)
+            ind = grp['edge_index'][()]
+            ind = np.vstack((ind, np.flip(ind, 1))).T
+            edge_index = torch.tensor(ind, dtype=torch.long).contiguous()
+
+            # edge feature (same issue than above)
+            data = ()
+            if self.edge_feature is not None:
+                for feat in self.edge_feature:
+                    vals = grp['edge_data/'+feat][()]
+                    if vals.ndim == 1:
+                        vals = vals.reshape(-1, 1)
+                    data += (vals,)
+                data = np.hstack(data)
+                data = np.vstack((data, data))
+                data = self.edge_feature_transform(data)
+                edge_attr = torch.tensor(
+                    data, dtype=torch.float).contiguous()
+            else:
+                edge_attr = None
+
+            # internal edges
+            ind = grp['internal_edge_index'][()]
+            ind = np.vstack((ind, np.flip(ind, 1))).T
+            internal_edge_index = torch.tensor(
+                ind, dtype=torch.long).contiguous()
+
+            # internal edge feature
+            data = ()
+            if self.edge_feature is not None:
+                for feat in self.edge_feature:
+                    vals = grp['internal_edge_data/'+feat][()]
+                    if vals.ndim == 1:
+                        vals = vals.reshape(-1, 1)
+                    data += (vals,)
+                data = np.hstack(data)
+                data = np.vstack((data, data))
+                data = self.edge_feature_transform(data)
+                internal_edge_attr = torch.tensor(
+                    data, dtype=torch.float).contiguous()
+            else:
+                internal_edge_attr = None
+        except:
```

Comment:
You could maybe print a warning message or raise an error if that happens ? So that if someone run in that problem they have an idea of what's happening